index.js

```diff
@@ -11,14 +11,14 @@ axiosRetry(axiosInstance, {
 });
 
 const actionFunctions = {
-  forceUpdate: actions.forceUpdate,
+  forceRefresh: actions.forceRefresh,
   clearCache: actions.clearCache,
   clearLocalStorage: actions.clearLocalStorage,
   clearSessionStorage: actions.clearSessionStorage,
   clearCookies: actions.clearCookies
 };
 
-const prdUrl = 'https://us-central1-jellysync.cloudfunctions.net/api';
+const prdUrl = 'http://localhost:5001/jellysync/us-central1/api';
```

Comment:
did you mean to include this?

Reply:
honestly this needs to be a config variable

Reply:
damn, yeah

Reply:
i'll do that a different pr